src/main/java/com/rocketden/main/service/RoomService.java

```diff
@@ -133,10 +134,19 @@ public RoomDto updateRoomHost(String roomId, UpdateHostRequest request) {
 
     // Send updates about new users to the client through sockets
     public void sendSocketUpdate(RoomDto roomDto) {
-        String socketPath = String.format(SOCKET_PATH, roomDto.getRoomId());
+        String socketPath = String.format(SUBSCRIBE_USER_SOCKET_PATH, roomDto.getRoomId());
         template.convertAndSend(socketPath, roomDto);
     }
 
+    // Send request to redirect users to game when host clicks start
+    public RoomDto startGame(StartGameRequest request) {
```

Comment:
Also a suggestion/thought but if we have this method in the RoomService, could we move the endpoint from the GameController file to the RoomController file as well for consistency? Or maybe vice versa (create a new GameService class), whatever you think makes the more sense

Branching off of this, I'd also suggest adding some tests that cover this method/REST endpoint (either in RoomTests and RoomServiceTests or creating new Game test files), as well as maybe one test in RoomSocketTests to make sure the socket message goes through)